pinot-common/src/main/proto/worker.proto

```diff
@@ -70,14 +70,15 @@ message QueryResponse {
 
 message StagePlan {
   int32 stageId = 1;
-  string virtualAddress = 2;
-  StageNode stageRoot = 3;
-  StageMetadata stageMetadata = 4;
+  StageNode stageRoot = 2;
+  StageMetadata stageMetadata = 3;
 }
 
 message StageMetadata {
   repeated WorkerMetadata workerMetadata = 1;
   map<string, string> customProperty = 2;
+  QueryServerInstance queryServerInstance = 3;
+  repeated int32 workerIds = 4;
```

Comment:
why do we need these 2? 

- queryServerInstance is not useful as when data is being sent. hostname and server port is already knew to the broker and channel is being selected. mailbox port is known to that particular server on its own JVM
- workerId list is actually part of the repeated WorkerMetadata (workerID is part of the virtualAddress)